packages/synthetic-chain/src/cli/doctor.ts

```diff
@@ -35,7 +35,11 @@ const fixupProposal = (proposal: ProposalInfo) => {
 
       // refresh install
       execSync('rm -rf node_modules', { cwd: proposalPath });
-      execSync('yarn install', { cwd: proposalPath });
+      // install to update yarn.lock and get importable typed modules but
+      // skip building because the proposal never runs on the local filesystem.
```

Comment:
my mental model is missing something...

What provides the binaries when the proposal runs?

Reply:
`node_modules` in the Docker image. It's already there. I had hoped that the mount would only be an overlay, but that was wrong. With mounting there's no way around running `yarn install` inside the container. I'll try to automate that so the developer doesn't have to remember/know.